frontend/ui/network/manager.lua

```diff
@@ -165,6 +165,7 @@ function NetworkMgr:setHTTPProxy(proxy)
 end
 
 function NetworkMgr:getWifiMenuTable()
+    if not Device:isAndroid() then
```

Comment:
Is this weird indentation here intentional? (If so I'd like to hear the rationale before saying it doesn't align with the coding style. :-P)

Reply:
intentional: lazyness porting from other branch. Will fix!

Reply:
Fixed!